build.sbt

```diff
@@ -221,39 +188,7 @@ lazy val shadedAsm = project.
     autoScalaLibrary := false,
     exportJars := true,
     packageBin in Compile := (assembly in asmDeps).value,
-    releaseVersion := { ver => ver },
-    bazelCustomBuild := BazelString(
-      """
-        |load(
-        |  '@io_bazel_rules_scala//scala:scala.bzl',
-        |  'scala_binary',
-        |  'scala_library',
-        |  'scala_test'
-        |)
-        |
-        |load(
-        |    "@com_github_johnynek_bazel_jar_jar//:jar_jar.bzl",
-        |    "jar_jar"
-        |)
-        |
-        |jar_jar(
-        |    name = "asmShaded",
-        |    input_jar = "@org_ow2_asm_asm//jar",
-        |    rules = "shade_rule",
-        |    visibility = [
-        |      '//visibility:public',
-        |    ]
-        |)
-        |
-        |jar_jar(
-        |    name = "asmTreeShaded",
-        |    input_jar = "@org_ow2_asm_asm_tree//jar",
-        |    rules = "shade_rule",
-        |    visibility = [
-        |      '//visibility:public',
-        |    ],
-        |)
-      """.stripMargin)
+    releaseVersion := { ver => ver }
```

Comment:
I don't know sbt well enough but could this be used to avoid us having to hand-modify the `version` all the time?